src/main/ngapp/accio/src/app/editor/editor.component.ts

```diff
@@ -855,7 +887,7 @@ export class EditorComponent implements OnInit {
   
   /** Calls the undo/redo 'do' function with paintedMask: Set of imageData indexes. */
   async doMaskActionPaint(): Promise<void> {
-    let paintedMask = await this.maskControllerPaint();
+    let paintedMask = await this.getPaintedSet();
```

Comment:
Is `await` needed now that `getPaintedSet()` isn't `async`?

Reply:
Chance pointed out that this function, or one that also called getpaintedset() didn't work on his end so I added that it returns a promise.